### DISCUSSION TITLE
表格行内编辑，动态更换组件如何实现

### DISCUSSION THREAD

**Discussion by IvesQing** [vbenjs/vue-vben-admin#3132] [General]
### Subject of the feature

 业务中有需求，假设表格有5个字段，[filed1,filed2,filed3,filed4,firld5]，其中filed3是下拉框组件，filed4字段的组件根据filed3的选择内容变更，当选择1,2,3时候是数字框，当选择4,5,6的时候，是选择框

### Problem

后端做前端，Vben帮了大忙，感谢诸位大佬。
遇到这种需求，该如何实现那。

### Expected behaviour

What should happen? Please describe the desired behaviour.

### Alternatives

What are the alternative solutions? Please describe what else you have considered?

**Comment by DesignHhuang** [comment]
其实和form的原理差不多，在监控filed3的onchange获取值后根据具体的值```setColumns```更新列。可以参考form中的一些例子。
